src/data/events.js

```diff
@@ -11,55 +11,55 @@ function getDateTime(
 }
 
 const events = [
+	// {
+	// 	name: 'MinionHack',
+	// 	date: getDateTime(2023, 4, 20, 9, 30),
+	// 	location: 'Carnesdale Commons Palisades Room',
```

Comment:
I know this is a comment, but I JUST realized this is a typo and should say "Carnesale" lmao